python/src/solver_module.i

```diff
@@ -53,6 +53,7 @@
 %include ODESolver.i
 %include RungeKutta.i
 %include Fehlberg.i
+%include LsqSolver.i
```

Comment:
new classes must go to experimental_module.i instead (correct the CMakeLists accordingly)